src/logic/Utils.ts

```diff
@@ -354,16 +344,17 @@ export const initAlbumArtBasedColor = (scheme: ColourScheme) => {
   // and update the color scheme accordingly
   Spicetify.Player.addEventListener("songchange", async () => {
     await sleep(1000);
-    let albumArt: HTMLImageElement | null = document.querySelector(".main-image-image.cover-art-image");
+    const albumArt: HTMLImageElement | null = document.querySelector(".main-image-image.cover-art-image");
+    let albumArtSrc: string | undefined = albumArt?.src;
```

Comment:
Ditto. Looks much cleaner and faster also.
```suggestion
    const albumArt = Spicetify.Player.data?.track?.metadata?.image_xlarge_url;
```